include/openssl/rand_drbg.h

```diff
@@ -17,7 +17,12 @@
 /* In CTR mode, disable derivation function ctr_df */
 # define RAND_DRBG_FLAG_CTR_NO_DF            0x1
 
-/* A logical OR of all used flag bits (currently there is only one) */
+/*
+ * A logical OR of all used flag bits
+ *
+ * Don't use this flag, it is intended for internal use only and will be
+ * removed from the public header in the next major release.
+ * /
```

Comment:
Would it make more sense to wrapper this and the define above with an deprecated #ifdef?

Reply:
Scratch the prior define.  It is necessary.

Reply:
Hmmm, how do you mean that? The flag _is_ used internally and that will not change. Only the define will be moved from the public header into a private one or to drbg_lib.c, see https://github.com/openssl/openssl/issues/7182#issuecomment-420436254.

Reply:
Ok, one could deprecate it and use a slightly renamed constant internally, as @mattcaswell suggested in https://github.com/openssl/openssl/issues/7182#issuecomment-420442566. But is it worth the effort?

Reply:
Ok, convinced. I'll do it and use your name suggestion.  But it's too late in the night now for me to scratch my head whether the internal define should go into drbg_lib.c only or into an internal header...

Reply:
I'm happy to grab this one and run with it.

Reply:
I'd go for `drbg_lib.c` since it isn't used anywhere else.
It also doesn't need to be a `#define`.  `const unsigned int`?
The name could then be kept but in lower case.